xbmc/platform/win32/WIN32Util.cpp

```diff
@@ -1191,6 +1191,66 @@ bool CWIN32Util::SetThreadLocalLocale(bool enable /* = true */)
   return _configthreadlocale(param) != -1;
 }
 
+std::wstring CWIN32Util::GetDisplayFriendlyName(const std::wstring& GdiDeviceName)
+{
+#ifdef TARGET_WINDOWS_STORE
+  // Not supported - not implemented yet
+  return std::wstring();
+#else
+
+  uint32_t pathCount = 0;
+  uint32_t modeCount = 0;
+  std::vector<DISPLAYCONFIG_PATH_INFO> paths = {};
+  std::vector<DISPLAYCONFIG_MODE_INFO> modes = {};
```

Comment:
```suggestion
  std::vector<DISPLAYCONFIG_MODE_INFO> modes;
```
vectors are already zero initialized at creation.

Reply:
sure